life_expectancy/tests/test_loading.py

```diff
@@ -0,0 +1,21 @@
+"""Tests for the loading module"""
+import pandas as pd
+from unittest import mock
+from life_expectancy.data_loading import load_data
+from life_expectancy.data_loading import save_data
+
+def test_load_data(eu_life_expectancy_raw_csv):
+
+    """Test for the load_data function"""
+    filepath = "life_expectancy/data/eu_life_expectancy_raw.tsv"
```

Comment:
Instead of consuming from the data directory, either read the fixture or use a patch to replace the return of the pd.read_csv for the fixture.

Reply:
mock_read_csv.return_value = A TUA FIXTURE

mock_read_csv.assert_called_once_with(...)